### PR TITLE
initial setup for prism deployment in Kyrgyzstan

### PR DESCRIPTION
**Pull request by wadhwamatic**
Initial setup for prism deployment in Kyrgyzstan which will use a combination of the Kyrgyzstan data cube and WFP's open data cube as primary data sources

### CODE REVIEW

**Comment by JorgeMartinezG** [comment]
@wadhwamatic Tests are failing. It looks like it is related to the default country changed here: https://github.com/WFP-VAM/prism-frontend/pull/217/files#diff-198a1431d7c5e26ea78bc6e54b34b54e6f8ab342dd48ea11013877e8466a87c5R42

**Comment by wadhwamatic** [comment]
> @wadhwamatic Tests are failing. It looks like it is related to the default country changed here: https://github.com/WFP-VAM/prism-frontend/pull/217/files#diff-198a1431d7c5e26ea78bc6e54b34b54e6f8ab342dd48ea11013877e8466a87c5R42

Thanks for catching that. I'm not sure why that change would break anything. I rolled it back to Mongolia as the default. Please check again

File: src/config/index.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ import {
 } from './mozambique';
 
 import { myanmarConfig, myanmarRawLayers, myanmarRawTables } from './myanmar';
+import {
```
Comment (ericboucher) [inline src/config/index.ts:22]: let's keep countries in alphabetical order when possible :)

File: src/config/kyrgyzstan/tables.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "animal_deaths": {
+    "title": "Animal deaths due to disease or natural hazards",
+    "table": "../data/myanmar/tables/NSO_Animal_Deaths_Annual_Admin1.csv",
```
Comment (ericboucher) [inline src/config/kyrgyzstan/tables.json:4]: myanmar?

**Review by ericboucher — APPROVED** [review]
Looks good. Minor comments

**Pull request merged**